src/matvis/cpu/cpu.py

```diff
@@ -81,6 +82,13 @@ def simulate(
         Either a 2D array, shape ``(Npairs, 2)``, or list of 2-tuples of ints, with
         the list of antenna-pairs to return as visibilities (all feed-pairs are always
         calculated). If None, all feed-pairs are returned.
+        matsets  :: array_like, optional
+                 A list of containing a set of 2-tuples of numpy arrays.  Each entry in the list
+                 corresponds to a different sub-matrix.  The first element of the ith tuple lists the rows of Z corresponding
+                 to the rows of the ith sub-matrix, and the second tuple element contains the list of columns.  In the case
+                 of vector-vector loop, the number of tuples is equal to Npairs and each element of the tuple contains only
+                 one int, such that the sub-matrices contain only one element each.  Outer dimension has to be a list because
+                 numpy doesn't like inhomogeneous arrays.
```

Comment:
try to follow the formatting of the other parameters -- the formatting has to be right for the documentation to compile

Reply:
I think I fixed the formatting, most of the issue was tabs vs. spaces.